bot/modules/lbrylink.js

```diff
@@ -12,31 +12,52 @@ exports.lbrylink = function(bot, msg, suffix) {
     var link = msg.content.includes("lbry://");
     if (link) {
       var text = msg.content.replace("lbry://", "https://open.lbry.io/");
-      var message = text.match(/\bhttps?:\/\/\S+/gi);
+      var message = text.match(/\bhttps?:\/\/\S+/gi).toString();
+      var check1 = message.includes(">");
       if (ResponseDebug == "true") {
         console.log("Link = " + link);
         console.log("text = " + text);
         console.log("message = " + message);
+        console.log("check1 = " + check1);
       }
-      if (text === "https://open.lbry.io/") {
+      if (message == "https://open.lbry.io/") {
```

Comment:
please if you can, use `===` instead of `==`. that ensures that the type is right as well, not only the value

Reply:
if i change then to equal signs the checks dont work properly